test/Microsoft.Azure.WebJobs.Host.FunctionalTests/DataBindingFunctionalTests.cs

```diff
@@ -51,6 +51,23 @@ public void BindNonStringableParameter_FailsIndexing()
                 exception.Message);
         }
 
+        [Fact]
+        public async Task FSharpTasks_AreCorrectlyIndexed()
```

Comment:
This test doesn't seem to belong in this test suite (DataBindingFunctionalTests). Seems like it belongs in FunctionIndexerTests